tests/bench/beam_common.py

```
@@ -0,0 +1,643 @@
+            max_tokens=2000,
+            temperature=0.0,
+            messages=cast(Any, messages),  # type: ignore[arg-type]
+            tools=cast(Any, [tool_definition]),  # type: ignore[arg-type]
+            tool_choice={
+                "type": "function",
+                "function": {"name": "evaluate_response"},
+            },
+        )
+
+        if not response.choices or not response.choices[0].message:
+            raise ValueError("OpenRouter returned empty response")
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)